lib/runtime.js

```diff
@@ -1,5 +1,6 @@
 var Entry = require("./entry.js").Entry;
 var utils = require("./utils.js");
+var dynRequire = module.require || __non_webpack_require__;
 
 exports.enable = function (Module) {
```

Comment:
This
```js
var dynRequire = module.require || __non_webpack_require__
``
is transformed into
```js
var dynRequire = module.require || require
```
Since the webpack bundled form doesn't have `module.require` it falls back to the real `require`.